tests/visible.lua

```diff
@@ -0,0 +1,165 @@
+local lu = require("luaunit")
+local vec = require("vector")
+local vis = require("visible")
+
+
+function testOrientation()
```

Comment:
Thanks for adding tests. The tests for vector functions should have their own file, tests/vector.lua